pavelib/assets.py

```diff
@@ -213,12 +312,105 @@ def compile_coffeescript(*files):
 @no_help
 @cmdopts([
     ('system=', 's', 'The system to compile sass for (defaults to all)'),
+    ('themes_dir=', '-td', 'The themes dir containing all themes (defaults to None)'),
+    ('themes=', '-t', 'The theme to compile sass for (defaults to None)'),
     ('debug', 'd', 'Debug mode'),
     ('force', '', 'Force full compilation'),
 ])
 def compile_sass(options):
     """
-    Compile Sass to CSS.
+    Compile Sass to CSS. following is a list of some possible ways to use this command.
+
+    Command:
+        1. paver compile_sass
+    Description:
+        compile sass files for both lms and cms
+
+    Command:
+        1. paver compile_sass --themes_dir=/edx/app/edxapp/edx-platform/themes --themes=red-theme
```

Comment:
Why do three of these commands start with ``1.`` and one doesn't? I'd remove the number as it seems confusing.